### PR TITLE
raftstore/store: add start time

### PR DESCRIPTION
**Pull request by huachaohuang**
To monitor store uptime.

Closes https://github.com/pingcap/tikv/issues/1207

### CODE REVIEW

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -267,6 +270,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             snap_mgr: mgr,
             raft_metrics: RaftMetrics::default(),
             tag: tag,
+            start_time: time::get_time(),
```
Comment (huachaohuang) [inline src/raftstore/store/store.rs]: I can't find a way to get the unix timestamp using std time, any suggestion?

**Comment by siddontang** [comment]
LGTM

**Comment by huachaohuang** [comment]
PTAL @zhangjinpeng1987

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**